content/03.categorize.md

```diff
@@ -231,6 +231,58 @@ a need for better pathologist-computer interaction techniques that will allow
 combining the power of deep learning methods with human expertise and lead to
 better-informed decisions for patient treatment and care.
 
+### Text applications in healthcare
+
+Due to the rapid growth of scholarly publications and electronic medical/health records, biomedical text mining has become increasingly important in recent years.
+The main tasks in biological and clinical text mining include, but are not limited to, named entity recognition, relation/event extraction, and information retrieval (Figure @fig:biotm).
+
+![Deep learning applications, tasks and models based on NLP perspectives.](images/biotm.png){#fig:biotm width="100%"}
+
+Machine learning is widely used in many of aforementioned tasks where engineering the optimal set of features is a must.
+However it is often difficult to know what features should be extracted, even with domain knowledge.
+Deep learning addresses this challenge by enabling "computers to build complex
+concepts out of simpler ones [@tag:goodfellow2016deep]".
+Moreover, deep learning has shown competitive performance compared to traditional ML methods.
+As a result, we have observed growing interests of using deep learning in biomedical text mining applications in recent years.
+In this section, we summarize recent studies in text mining with various deep learning methods.
+We categorize the body of work by the application domain/text genre (biomedical literature vs. clinical notes) and by the actual task (e.g. concept or relation extraction).
+
+In biomedical text mining, named entity recognition (NER) is a task of identifying text spans that refer to a biological concept of a specific class, such as disease or chemical, in a controlled vocabulary or ontology.
+NER is of significant importance as it is often needed as a first step in many complex text mining systems.
+The current state-of-the-art methods typically reformulate the task as a sequence labeling problem and use conditional random fields [@doi:10.1093/bioinformatics/btw343; @doi:10.1093/bioinformatics/btt156; @doi:10.1093/bioinformatics/btt474].
+In recent years, word embeddings that contain rich latent semantic information of words have been widely used to improve the NER performance.
```

Comment:
pinging @gwaygenomics just because we've talked about adding something on latent spaces / etc to the discussion. This is a nice addition & should be mentioned there if it goes ahead.

Reply:
Wanted to close the loop on this. @gwaygenomics proposed a new PR https://github.com/greenelab/deep-review/pull/718 that talks about latent spaces now in the discussion.

Reply:
@luzh2 - feel free to comment on #718 if you think there are improvements to be made.